### PR TITLE
SCIM Endpoints - beta

### PR DESCRIPTION
**Pull request by KellieHawks**
# Pull Request/Issue Resolution

Closes #**ISSUE_NUMBER_HERE**

#### Is this change associated with a Braze feature/product release?
- [ ] Yes (**Insert Feature Release Date Here**)
- [ ] No

---

<details>
<summary>✔️ Pull Request Checklist</summary>
<br>

- [ ] Check that all links work.
- [ ] Ensure you have completed [our Contributors License Agreement](https://www.braze.com/docs/cla/).
- [ ] Tag @Timothy-Kim and @KellieHawks as a reviewer when your work is **done and ready to be reviewed for merge**. Are you an internal product manager? Reference the interal reviewing chart to tag the appropriate reviewer.
- [ ] Tag others as reviewers as necessary.
- [ ] If you have modified any links, be sure to add redirects to `assets` > `js` > `broken_redirect_list.js`

</details>

<details>
<summary>⭐ Helpful Wiki Shortcuts</summary>
<br>

- [Writing Style Guide](https://docs.google.com/document/d/e/2PACX-1vTluyDFO3ZEV7V6VvhXE4As_hSFwmnFFdU9g6_TrAYTgH1QmbRoEDDdn5GzKAB9vdBbIdyiFdoaJcNk/pub)
- [Image Style Guide](https://docs.google.com/document/d/e/2PACX-1vRJSkwcjmjrTfLDagZccLpOMMyh5NN5SXRZSjz12cRAHbX4OrUmhvCmYpf_p5YB-9r4_jSOQLkicQIH/pub)
- [Styling Test Page](https://www.braze.com/docs/home/styling_test_page/)

</details>

<details>
<summary>❗ ATTN: For PR Reviewers</summary>
<br>

- [ ] Read our [Reviewing a PR page](https://github.com/Appboy/braze-docs/wiki/Reviewing-a-PR) for more on our reviewing suggestions.
- [ ] Read our [Previewing Documentation page](https://github.com/braze-inc/braze-docs/wiki/Previewing-and-Testing-Documentation) to see how to check the deployment.
  - [ ] Preview all changes in the linked Vercel environment by clicking the preview link in the vercel-bot comment in your PR.
</details>

<details>
<summary>❗ ATTN: Internal Reviewing Chart </summary>
<br>
<b>Work at Braze and not sure who to tag for review?</b> <br>Before tagging @timothy-kim or @KellieHawks for a general review, reference the following chart to see if a specific product vertical/reviewer applies to your pull request.
<br><br>
<table>
<tr>
    <td><b>Reviewer</b></td>
    <td><b>Product Vertical</b></td>
  </tr>
  <tr>
    <td>@Timothy-Kim</td>
    <td>Data Infrastructure<br>Platform Infrastructure<br>Quality Infrastructure</td>
  </tr>
  <tr>
    <td>@kelliehawks</td>
    <td>Internal Tools<br>Product Partnerships<br>SDKs<br>SMS</td>
  </tr>
  <tr>
    <td>@bre-fitzgerald</td>
    <td>Channels<br>FIX<br>Intelligence<br>Reporting<br>SMB</td>
  </tr>
  <tr>
    <td>@lydia-xie</td>
    <td>Email (Composition and Infrastructure)<br>Ingestion<br>Messaging</td>
  </tr>
</table>
</details>

### CODE REVIEW

File: _docs/_hidden/private_betas/scim.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+description: "This landing page lists the Braze SCIM Endpoints."
+page_type: landing
+
+guide_top_header: "SCIM Endpoints"
+guide_top_text: "Use the Braze SCIM Endpoints to manage automated user provisioning."
+
+guide_featured_title: "Braze SCIM Endpoints"
+guide_featured_list:
+  - name: "POST: Create Dashboard Developer Account"
+    link: /docs/scim/post/
```
Comment (bre-fitzgerald) [inline _docs/_hidden/private_betas/scim.md:19]: @KellieHawks tried a few times but none of the links to the endpoints on this page are working for me 🤔

Comment (KellieHawks) [reply _docs/_hidden/private_betas/scim.md]: Fixed!

File: _docs/_hidden/private_betas/scim/post_create_developer_account.md [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+```
+
+## Request parameters
+
+| Parameter | Required | Data type | Description |
+| --------- | -------- | --------- | ----------- |
+| Schemas | Required | Array of strings | Expected SCIM 2.0 schema name for user object. |
+| `id` | Required | String | The developer's email address |
+| `username` | Required | String | The username that the developer will need to log into Braze (usually the same as email address) |
+| `name` | Required | JSON object | This object contains the develoepr's first name and last name |
```
Comment (bre-fitzgerald) [inline _docs/_hidden/private_betas/scim/post_create_developer_account.md:50]:
```suggestion
| `name` | Required | JSON object | This object contains the developer's first name and last name |
```

File: _docs/_hidden/private_betas/scim/put_update_existing_developer_account.md [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+```
+
+## Request parameters
+
+| Parameter | Required | Data type | Description |
+| --------- | -------- | --------- | ----------- |
+| Schemas | Required | Array of strings | Expected SCIM 2.0 schema name for user object. |
+| `id` | Required | String | The developer's email address |
+| `username` | Required | String | The username that the developer will need to log into Braze (usually the same as email address) |
+| `name` | Required | JSON object | This object contains the develoepr's first name and last name |
```
Comment (bre-fitzgerald) [inline _docs/_hidden/private_betas/scim/put_update_existing_developer_account.md:49]:
```suggestion
| `name` | Required | JSON object | This object contains the developer's first name and last name |
```

**Review by bre-fitzgerald — CHANGES_REQUESTED** [review]

**Review by bre-fitzgerald — APPROVED** [review]
Just caught a few typos i missed last round, looks good to go!

**Pull request merged**